packages/curve-ui-kit/src/shared/ui/CopyIconButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { useSwitch } from '@ui-kit/hooks/useSwitch'
+import Tooltip from '@mui/material/Tooltip'
+import IconButton from '@mui/material/IconButton'
+import { CopyIcon } from '@ui-kit/shared/icons/CopyIcon'
+import Snackbar from '@mui/material/Snackbar'
+import { Duration } from '@ui-kit/themes/design/0_primitives'
+import Alert from '@mui/material/Alert'
+import AlertTitle from '@mui/material/AlertTitle'
+
+export function CopyIconButton({
```

Comment:
I guess you could've also put this inside `CopyIcon.tsx`, like you did with `HeartIcon.tsx`. I don't mind either way though

Reply:
I dont think the `icons` folder would be appropriate for a button